src/connective.py

```diff
@@ -16,6 +16,7 @@ def on_ready():
         "target_connectives",
         {
             "range_connective": range_connectives,
+            "range_column": {"vertical": "rangeColumn"},
```

Comment:
maybe
```suggestion
            "range_column": {"vertical": "verticalRange"},
```

I'm almost tempted to have this one in `modifiers` or a separate `rangeModifiers`, but I don't feel super strongly.  It's not actually a connective, so feels weird there.  I recognize that today we only use it as a modifier on a connective, but I'm tempted to allow it as a modifier before "tail", because "tail" effectively constructs a range.

This comment is the only one I'd give careful thought to; the other ones are easier to change down the road (tho they do require a cursorless api change), but this one we lock ourselves into via the settings file.

Reply:
Moving it to modifiers and renaming it verticalRange is fine with me